Battery-Commander.Web/Views/Evaluations/List.cshtml

```diff
@@ -52,6 +52,7 @@
                 <th>@Html.DisplayNameFor(_ => _.Evaluations.FirstOrDefault().Ratee.Rank)</th>
                 <th data-priority="1">@Html.DisplayNameFor(_ => _.Evaluations.FirstOrDefault().Ratee.LastName)</th>
                 <th>@Html.DisplayNameFor(_ => _.Evaluations.FirstOrDefault().Ratee.FirstName)</th>
+                <th>@Html.DisplayNameFor(_ => _.Soldiers.FirstOrDefault().Unit)</th>
```

Comment:
Make sure you add it to the table body as well, this is just for the header.

Reply:
@mattgwagner  added to the body should be gtg now